ai-ml/gke-ray/rayserve/stable-diffusion/ray-service-tpu.yaml

```diff
@@ -31,14 +31,14 @@ spec:
             - -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
             - fastapi
   rayClusterConfig:
-    rayVersion: '2.9.0'
+    rayVersion: '2.36.1'
     headGroupSpec:
       rayStartParams: {}
       template:
         spec:
           containers:
           - name: ray-head
-            image: rayproject/ray-ml:2.9.0-py310
+            image: rayproject/ray-ml:2.36.1
```

Comment:
same comment here I don't think this tag exists

Reply:
updated